whitePages/whitePages-war/src/main/java/org/silverpeas/components/whitepages/control/WhitePagesSessionController.java

```diff
@@ -926,32 +906,32 @@ public void confirmFieldsChoice(String[] fields) {
     return ids;
   }
 
-  public List<Card> getSearchResult(String query, SearchContext pdcContext,
+  public List<Card> getSearchResult(String query, String taxonomyPosition,
       Map<String, String> xmlFields, List<FieldDescription> fieldsQuery) {
     List<Card> cards = new ArrayList<>();
-    Collection<GlobalSilverContent> contents = null;
+    List<SearchResult> results = null;
 
     try {
       PublicationTemplate template = getTemplate(getComponentId());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "template" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1481)